registrations/models.py

```diff
@@ -103,6 +103,29 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
 
+class ObjectsWithoutSoftDeletedManager(SerializableMixin.SerializableManager):
+    def get_queryset(self):
+        return super().get_queryset().filter(deleted=False)
+
+
+class SoftDeletableBaseModel(models.Model):
+    deleted = models.BooleanField(default=False)
+
+    objects = ObjectsWithoutSoftDeletedManager()
+    all_objects = models.Manager()
+
+    def soft_delete(self):
```

Comment:
I believe (not entirely sure) that auto_now fields will not get updated when using update_fields unless they are included, so here could consider doing some hasattr introspection and conditionally also updating `last_modified_time` ?

Reply:
(and last_modified_by  if available?)